Menus/BuyMenuGUI.cpp

```diff
@@ -1320,14 +1320,14 @@ void BuyMenuGUI::Update()
                 if (craftMaxMass == 0) {
                     description += "\nNO CARGO SPACE!";
                 } else if (craftMaxMass > 0) {
-                    description += "\nMax Mass: " + RoundFloatToPrecision(craftMaxMass, 1) + " kg";
+                    description += "\nMax Mass: " + RoundFloatToPrecision(craftMaxMass, craftMaxMass < 9.95F ? 1 : 0) + " kg";
```

Comment:
I think it would be better to bump this up an order of magnitude, so it'll show a decimal place if it's < 100. I find it looks weird to have `9.0 kg` on one thing, and `10 kg` on another, but using 100 as the dividing line

1. You're less likely to have things on both sides of that in 1 category. I.e. most actors weigh > 100 and most guns or bombs weigh < 100, so it feels more consistent

2. One could argue a half kilo still matters at 10 kg, but it definitely doesn't matter at 100 kg


If you have no issue with this, I'm happy to do the code change.

Reply:
You're right that 0.1 is less nice that 0.5 but I otherwise disagree with you. It's all a visual thing and, visually I strongly stand by what I said before - just open up the buy menu and scroll through guns, it looks inconsistent and bad as-is.
If anything you could round 50.1 to the nearest 0.5 and call it a day, anyway.